source/manual/zendesk-smartsurvey-feedback-data-pipeline.html.md.erb

```diff
@@ -0,0 +1,145 @@
+---
+owner_slack: "#govuk-data-labs"
+title: Zendesk and SmartSurvey feedback data pipeline
+section: Data Community Documentation
+layout: manual_layout
+parent: "/manual.html"
+---
+
+GOV.UK users can select a feedback question at the bottom of a GOV.UK webpage to provide feedback.
+
+There are 2 feedback questions. If the feedback question is:
+
+- “Is this page useful?”, the feedback data goes to SmartSurvey
+- “Is there something wrong with this page?”, the feedback data goes to Zendesk
```

Comment:
can we link to our Google Slides on here, what's the etiquette / rules on linking to files on our drive, given this is public?

Reply:
@mammykins we can link to a file on the drive as long as we say that it requires access and how to request that access. what is the link and how can a user ask for access to that link?

Reply:
Changed my mind, this is sufficient